bitnami/vault/templates/injector/deployment.yaml

```
@@ -0,0 +1,184 @@
+        app.kubernetes.io/component: injector
+        {{- if .Values.injector.podLabels }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.injector.podLabels "context" $) | nindent 8 }}
+        {{- end }}
+        {{- if .Values.commonLabels }}
+        {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 8 }}
+        {{- end }}
+    spec:
+      serviceAccountName: {{ template "vault.injector.serviceAccountName" . }}
+      {{- include "vault.imagePullSecrets" . | nindent 6 }}
+      {{- if .Values.injector.hostAliases }}
+      hostAliases: {{- include "common.tplvalues.render" (dict "value" .Values.injector.hostAliases "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.injector.affinity }}
+      affinity: {{- include "common.tplvalues.render" ( dict "value" .Values.injector.affinity "context" $) | nindent 8 }}
+      {{- else }}
+      affinity:
+        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.injector.podAffinityPreset "component" "injector" "context" $) | nindent 10 }}
+        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.injector.podAntiAffinityPreset "component" "injector" "context" $) | nindent 10 }}
+        nodeAffinity: {{- include "common.affinities.nodes" (dict "type" .Values.injector.nodeAffinityPreset.type "key" .Values.injector.nodeAffinityPreset.key "values" .Values.injector.nodeAffinityPreset.values) | nindent 10 }}
+      {{- end }}
+      {{- if .Values.injector.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" ( dict "value" .Values.injector.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.injector.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.injector.tolerations "context" .) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.injector.priorityClassName }}
+      priorityClassName: {{ .Values.injector.priorityClassName | quote }}
+      {{- end }}
+      {{- if .Values.injector.schedulerName }}
+      schedulerName: {{ .Values.injector.schedulerName | quote }}
+      {{- end }}
+      {{- if .Values.injector.topologySpreadConstraints }}
+      topologySpreadConstraints: {{- include "common.tplvalues.render" (dict "value" .Values.injector.topologySpreadConstraints "context" .) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.injector.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.injector.podSecurityContext "enabled" | toYaml | nindent 8 }}
+      {{- end }}
+      {{- if .Values.injector.terminationGracePeriodSeconds }}
+      terminationGracePeriodSeconds: {{ .Values.injector.terminationGracePeriodSeconds }}
+      {{- end }}
+      {{- if .Values.injector.initContainers }}
+      initContainers:
+        {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
```

There is a typo here:
```suggestion
        {{- include "common.tplvalues.render" (dict "value" .Values.injector.initContainers "context" $) | nindent 8 }}
```